packages/engine/tests/experiment/mod.rs

```diff
@@ -160,35 +163,55 @@ pub async fn run_test_suite(
             // Remove log file in case it's already existing
             let _ = fs::remove_file(&log_file_path);
 
-            let test_result = run_test(
-                experiment_type.clone(),
-                &project_path,
-                project_name.clone(),
-                output_folder,
-                language,
-                expected_outputs.len(),
-            )
-            .await;
-
-            let test_result = match test_result {
-                Ok(outputs) => outputs,
-                Err(err) => {
-                    match fs::read_to_string(&log_file_path) {
-                        Ok(log) => eprintln!("{log}"),
-                        Err(err) => eprintln!("Could not read {log_file_path:?}: {err}"),
+            let mut test_output = None;
+            for rust_log_env in &rust_log_envs {
+                let environment = std::env::vars();
+                std::env::set_var("RUST_LOG", &rust_log_env);
```

Comment:
I think we have to somehow manually pass in the log level. Using env-vars won't work as they're within the same process

Reply:
I added a command-line argument and use this instead.